src/Umbraco.Core/Persistence/Dtos/RelationTypeDto.cs

```diff
@@ -9,7 +9,7 @@ namespace Umbraco.Core.Persistence.Dtos
     [ExplicitColumns]
     internal class RelationTypeDto
     {
-        public const int NodeIdSeed = 4;
+        public const int NodeIdSeed = 10;
```

Comment:
That's necessary because we are adding more relation dtos during install which exceed the default Seed value. If we don't change this, then the next relation type that is added is going to throw an exception since the Seed will have been configured lower than what is now available. It's one of those annoying 'gotchas' if we are adding more data during the default installation to watch out for.